content/applications/websites/website/reporting/link_tracker.rst

```diff
@@ -2,55 +2,75 @@
+driving traffic and conversions.
 
-Go to :menuselection:`Website --> Configuration --> Settings` and activate *Link Trackers*.
+Website visibility
+------------------
 
-.. image:: link_tracker/enable_link_tracker.png
-   :align: center
-   :alt: View of Website settings page emphasizing the link trackers field in Odoo Website
+The :abbr:`UTM (Urchin Tracking Module)` parameters allow you to display some site elements only for
+specific audiences. To achieve this, click the :guilabel:`Edit` button on your website, select the
+building block for which you wish to adjust the visibility, go to the :guilabel:`Customize` tab,
+scroll down to :guilabel:`Visibility`, and click :guilabel:`Conditionally`.
 
-Set up traceable URLs
----------------------
+.. image:: link_tracker/conditional-visibility.png
+   :alt: Use the conditional visibility to display some site elements only for specific audiences.
 
-Go to :menuselection:`Website --> Go to website --> Promote --> Track this page`. Here, you are able
-to get a specific tracked URL based on the campaign, medium, and source being used.
+For each attribute available in the `Visibility <https://www.odoo.com/documentation/17.0/applications/websites/website/web_design/building_blocks.html#visibility>`_ section, you can choose
+:guilabel:`Visible for` or :guilabel:`Hidden for` and choose the record you want from the dropdown
+list.
 
-.. image:: link_tracker/link_tracker_fields.png
-   :align: center
-   :alt: View of the link tracker fields for Odoo Website
+Your tracked links
+==================
 
-- **URL**: url of the page you want to track (e.g. the home page or a product's page).
-- **Campaign**: context of your link (e.g. a special promotion).
-- **Medium**: channel used to share (deliver) your link (e.g. an email or a Facebook ad).
-- **Source**: platform where the traffic originates (e.g. Google or Twitter).
+To get an overview of your tracked links, go to :menuselection:`Website --> Site --> Link Tracker`,
+and scroll down to :guilabel:`Your tracked links` section.
 
-Now, click on *Get tracked link* to generate a URL that you can post or send by the source you have
-decided on.
+.. image:: link_tracker/your-tracked-links.png
+   :alt: Get an overview of all the links you track.
 
-Follow-up on tracked links
-==========================
+Statistics
+----------
 
-To look at statistics of your links, go to :menuselection:`Website --> Go to website --> Promote
---> Track this page`. Besides being able to see the *Most Clicked* and *Recently Used* links, you
-can also see complete statistics by clicking on *Stats*, including the number of clicks, and the
-country of origin for those clicks.
+Using tracked links provides the advantage of obtaining statistics to assess the performance of your
+campaign, medium, and source. This allows you to determine the number of customers who visited your
+website through the tracked link.
```

Comment:
```suggestion
```
Already mentioned in the intro, maybe just add "To measure the performance of the tracked links,..."